waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/client/TunnelingMetaStoreClientFactoryTest.java

```diff
@@ -76,4 +77,10 @@ public void newInstanceWithoutTunneling() throws Exception {
     verifyZeroInteractions(tunnelableFactorySupplier);
   }
 
+  @Test
+  public void newInstanceWithoutTunnelingNorSupplier() {
+    TunnelingMetaStoreClientFactory clientFactory = new TunnelingMetaStoreClientFactory();
+    clientFactory.newInstance(hiveConf, "test", 10);
```

Comment:
What are you testing here? Just that no exceptions are being thrown? (there are no assertions)

Reply:
This is supposed to test the first constructor of the class that doesn't need any parameters.

Reply:
Yes, but you're not doing anything after that, is there anything you can assert? Maybe based on the values you've passed in to check they're being used as expected?

Reply:
So far I have tried to get the configuration values from the object returned by `newInstance`, but it does not let me access config values: `For security reasons, the config key com.hotels.bdp.waggledance.client.ssh.localhost cannot be accessed`.
This happens for all the variables in the config.

Also, the client name and number of retries are passed to a private class within `DefaultMetaStoreClientFactory` so I don't think those are accessible either.

One thing that I can check for is that the object returned from `clientFactory.newInstace(...)` is not null. Is there anything else that I'm missing?